manifests/v/VMware/HorizonClient/8.7.0.31805/VMware.HorizonClient.locale.en-US.yaml

```
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-608.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: VMware.HorizonClient
+PackageVersion: 8.7.0.31805
+PackageLocale: en-US
+Publisher: VMware, Inc.
+PublisherUrl: https://www.vmware.com
+PublisherSupportUrl: https://www.vmware.com/support/horizon-view.html
+PrivacyUrl: https://www.vmware.com/help/privacy.html
+Author: VMware
+PackageName: VMware Horizon Client
+PackageUrl: https://www.vmware.com/products/horizon.html
+License: © 2022 VMware, Inc
```

```suggestion
License: Proprietary
```
A copyright isn't a license.